include/freerdp/server/cliprdr.h

```diff
@@ -68,6 +68,12 @@ struct _cliprdr_server_context
 	void* handle;
 	void* custom;
 
+	/* clipboard capabilities - set by server */
```

Comment:
@mfleisz am I mistaken or are the following values in the end the combination 
of what is supported by the server and the client?